src/PasswordIndicatorGrid.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+import "./PasswordIndicatorGrid.css";
+import { StrengthIndicator } from "./components/PasswordStrengthIndicator";
+
+function PasswordIndicatorGrid(props) {
+  let indicators = [
+    {
+      name: "Basic Indicator",
+      determineStrength: password => {
+        password = password || "";
+        const strength = Math.min(password.length, 10);
+        let description = "weak";
+        if (strength > 4 && strength < 7) {
+          description = "okay";
+        } else if (strength >= 7) {
+          description = "strong";
+        }
+
+        return { strength, description };
+      }
+    }
+  ];
+
+  indicators = [...indicators, ...indicators, ...indicators, ...indicators];
```

Comment:
I duplicated the "basic indicator" function four times so it would fill up the screen, no other reason for this line